OpenRA.Mods.Common/Widgets/Logic/MusicPlayerLogic.cs

```diff
@@ -36,29 +36,30 @@ public MusicPlayerLogic(Widget widget, Ruleset modRules, World world, Action onE
 			BuildMusicTable();
 
 			Func<bool> noMusic = () => !musicPlaylist.IsMusicAvailable;
+			Func<bool> disableControls = () => !musicPlaylist.IsMusicAvailable || musicPlaylist.AmbientMusic;
```

Comment:
I think my earlier comments must have confused you somewhere.  We can discuss this on IRC later tonight.